WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -30,3 +28,9 @@
     </li>
   <% end %>
 </ul>
+
+<script>
+  $('.date i').tooltip({
```

Comment:
How about a quick comment explaining why we can't use data-toggle here? I worry someone will see this and decide they can clean it pu.